### PR TITLE
Progress: Progress in Bottom Bar

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
Depends to https://github.com/Samsung/TAU/pull/1569

[Issue] https://github.com/Samsung/TAU/issues/1559
[Problem] Progress on Bottom Bar needs implemented
[Solution]
   - added new template in UIComponents
   - added new styles

[Screenshot]
image

image

image

### CODE REVIEW

File: src/css/profile/mobile/common/progress.less [hunk 1]
```diff
@@ -398,3 +398,80 @@ tau-progress {
...
+		}
+		.ui-progress-text {
+			font-size: 14 * @sp_base;
+			padding: 0;
+			margin: auto;
+		}
+		.ui-indeterminate-circle-medium {
+			width: 24 * @px_base;
+			height: 24 * @px_base;
+			margin-right: 8px;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/progress.less]: Use @px_base?

File: src/css/profile/mobile/common/progress.less [hunk 2]
```diff
@@ -398,3 +398,80 @@ tau-progress {
...
+			&::before, &::after {
+				width: (24/2 - 3) * @px_base;
+				height: (24/2 - 3) * @px_base;
+			}
+		}
+		.ui-progress-bar ~ .ui-progress-text {
+			padding-top: 3px;
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/progress.less]: ditto.

**Review by HunseopJeong — APPROVED** [review]
LGTM but need to fix minor issues

File: src/css/profile/mobile/common/progress.less (hunk 1 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/progress.less]: You are right of course :) . I'll fix it.

Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/progress.less]: Done.

File: src/css/profile/mobile/common/progress.less (hunk 2 above)
Comment (TomaszLukawskiSam) [reply src/css/profile/mobile/common/progress.less]: Done.

**Review by lbardeli — APPROVED** [review]

**Pull request merged**